### PR TITLE
Added my first outreachy blog post.

### PR DESCRIPTION
**Pull request by Burnleydev1**
I just started my outreachy internship with the OCaml community and would like to link to my first post which is an introductory blog post in the [OCaml.org](http://ocaml.org/) blog, thank you

### CODE REVIEW

**Comment by Burnleydev1** [comment]
Hello @pitag-ha this is an attempted pull request for my blog post addition,

File: data/planet/outreachy/first-week-at-outreachy.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Outreachy Blog #1: Introduce Yourself
```
Comment (sabine) [inline data/planet/outreachy/first-week-at-outreachy.md]:
```suggestion
title: Outreachy Blog \#1: Introduce Yourself
```

File: data/planet/outreachy/first-week-at-outreachy.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+description: To introduce myself and give my core values.
```
Comment (sabine) [inline data/planet/outreachy/first-week-at-outreachy.md]:
```suggestion
description: As part of the Outreachy internship program, I introduce myself and give my core values.
```

**Review by sabine — APPROVED** [review]

**Comment by Burnleydev1** [comment]
Hi @sabine, Thanks a lot for the review and corrections.

**Comment by sabine** [comment]
@Burnleydev1  I love seeing the blog posts from the Outreachy interns, so great that you all make the effort to write them! :+1: :slightly_smiling_face:

**Pull request merged**